### PR TITLE
Adding ATM9 Resource Pack Depot

### PR DESCRIPTION
**Pull request by TyBraniff**
Adding ATM9 Resource Pack Depot

### CODE REVIEW

File: assets/addon_browser/addons.conf [hunk 1]
```diff
@@ -614,4 +614,59 @@
...
+    apiVersion: ""
+    platforms: ["resourcepack"]
+    links: {
+      "other": "https://bmrp.bronzedeathsquad.com/mods/"
+      "other": "https://discord.com/channels/665868367416131594/1224815399401554013"
+    }
+  }
+  {
+    name: "Sophisticated Storage - Bluemap Resource Pack"
+    description: "A Resource pack to help Sophisticated Backpacks Mod load models and textures correctly with Bluemap"
```
Comment (TechnicJelle) [inline assets/addon_browser/addons.conf:663]:
```suggestion
    description: "A Resource pack to help Sophisticated Storage Mod load models and textures correctly with Bluemap"
```

**Review by TechnicJelle — CHANGES_REQUESTED** [review]
A couple general bits of feedback:
- Mentioning in the `name` that it's a "Bluemap Resource Pack" is kind of unnecessary, seeing as we're on the BlueMap wiki, and that it's already tagged as being a resourcepack.
- We discussed it internally, and we decided that links should go to either a dedicated page with more information about the resource (perhaps some contact info at least), or the direct download link itself. (So not a single page with just more download links to other "unrelated" things.)
If you could add some info about what that collection is, on the https://bmrp.bronzedeathsquad.com/mods/ webpage, then I think that would be very good.
- The Discord link is kind of unnecessary. People can join the discord and go to the `#3rd-party-support` channel themselves if they have issues with it. For those people, it might be nice if you mention that, and the username they can ping there on your depot website, for example.

**Comment by TyBraniff** [comment]
The Private distro was slapped up as a quick tool to just download my packs whenever I needed them, From pc to pc etc. I've since started releasing them on Github and Modrinth so the need for the private distro is not even needed anymore. This should give me direct links to point to for each mod.

As for mentioning ATM9 its not even close to time for that. It started out with that but currently mod to mod basis is better. Once I get more in depth with Ad Astra, Allthemodium, The Aether, Blueskies, and some of the other 430+ mods I'll feel more comfortable releasing a Modpack version that has all these packs wrapped into one nice resource pack for easy download/install. 

I'll redo my request for addons.conf in the next few days.

**Comment by TyBraniff** [comment]
I've added all the ones I've uploaded to Github/Modrinth and supplied the links to them.
I will be compiling more over the next few weeks but this should be enough for now, unless there's another way we want me to do this.

I will release an ATM9 Modpack resource pack when I have more confirmation on blocks that are missing/wrong with Ad Astra and Create.

**Comment by TBlueF** [comment]
Any update on this? Can this be merged? @TechnicJelle? ^^

**Comment by TechnicJelle** [comment]
Is TysFixes still necessary, or was everything from there merged into BlueMap itself?

**Comment by TBlueF** [comment]
I think that's all merged .. it would in theory still apply to older bluemap versions but .. yeah

**Comment by TechnicJelle** [comment]
I also see a few more fixerpacks on Ty's server than are in this list
I seem to remember this PR was a WIP, but I'm not 100% sure anymore _what_ was WIP about it, haha..
Do you remember, @TyBraniff ?

**Comment by TyBraniff** [comment]
> I also see a few more fixerpacks on Ty's server than are in this list I seem to remember this PR was a WIP, but I'm not 100% sure anymore _what_ was WIP about it, haha.. Do you remember, @TyBraniff ?

It was considered WIP because I was finding new blocks as I played. Currently I'm on a break from ATM9. I have made arrangements with another server owner recently to produce a few more bluemap resources packs. I will be starting those hopefully in the next week or so. 

Can confirm some of the fixes across my packs are still relavent for 5.0 builds of bluemaps since the server owner I'm working with needed some of my existing packs to fix textures. Notably this was for twilight forest.

**Pull request merged**